common/compositor/compositor.cpp

```diff
@@ -212,6 +213,21 @@ void Compositor::RestoreVideoDefaultColor(HWCColorControl /*color*/) {
   // TODO
 }
 
+void Compositor::SetVideoSharp(float value) {
+  lock_.lock();
+  sharp_ = value;
```

Comment:
I am wondering if instead of storing it here, should we push it all the way to CompositorThread and store it their ? Than when rendering media layers we can set to MediaRenderer. We can avoid passing this vai mediastate. Same with colors if possible

Reply:
Er, I think colors actually pass value via MediaState .

Reply:
k, if it varies per media layer than ya. But if not I dont think its needed via Media state